testsuite/tests/typing-modules/nondep_private_abbrev.ml

```diff
@@ -80,20 +80,20 @@ module IndirectPub : sig type t = [ `Foo of 'a ] as 'a end
    if we were unrolling the abbrev.  *)
 module IndirectPriv = I(struct end);;
 [%%expect{|
-module IndirectPriv : sig type t end
+module IndirectPriv : sig type t : immutable_data end
 |}]
 
 (* These two behave as though a functor was defined *)
 module DirectPrivEta =
   (functor (X : sig end) -> Priv(X))(struct end);;
 [%%expect{|
-module DirectPrivEta : sig type t end
+module DirectPrivEta : sig type t : immutable_data with t end
```

Comment:
That is not a very helpful kind. It seems sound, but we should leave a CR to examine the interaction between kinds and nondep. Urgh.

Reply:
Done.